pkg/agent/util/net_windows.go

```diff
@@ -579,3 +582,34 @@ func parseRoutes(routesStr string) ([]Route, error) {
 	}
 	return routes, nil
 }
+
+// GetVirtualInterfaceByName returns the virtual interface created by Windows when plugging the specified interface
+// on the HNSNetwork. If the virtual interface is not exist as well, the OVS bridge interface is returned. This function
```

Comment:
```suggestion
// on the HNSNetwork. If the virtual interface does not exist, the OVS bridge interface is returned. This function
```
And this function is not generic enough to be util either, maybe move the fallback logic to client side.